src/tests/test.py

```diff
@@ -28,9 +31,20 @@ async def before_after(coro, *args, **kwargs):
     return "<before>" + (await coro(*args, **kwargs)) + "<after>"
 
 
+def asyncio_run(awaitable):
+    """asyncio.run for Python 3.5"""
+    try:
+        aio_run = run
```

Comment:
How can this give an AttributeError? I would have expected the import logic to be here instead.

Reply:
You are right, I've updated PR.